deploy-service/common/src/main/java/com/pinterest/deployservice/whitelists/Whitelist.java

```diff
@@ -2,4 +2,5 @@
 
 public interface Whitelist {
```

Comment:
while we're here do you think we could update whitelist->allowlist?

Reply:
good point. will do

Reply:
decide to not do this re-factor at this time. this PR will focus on supporting the trusted URL. We can do the refactor in a later time.